tests/Model/Generated/Object/NotificationUrlTest.php

```diff
@@ -0,0 +1,296 @@
+<?php
+namespace bunq\test\Model\Object;
+
+use bunq\Exception\BunqException;
+use bunq\Model\Core\BunqModel;
+use bunq\Model\Generated\Endpoint\BunqMeTab;
+use bunq\Model\Generated\Endpoint\ChatMessageAnnouncement;
+use bunq\Model\Generated\Endpoint\DraftPayment;
+use bunq\Model\Generated\Endpoint\MasterCardAction;
+use bunq\Model\Generated\Endpoint\MonetaryAccountBank;
+use bunq\Model\Generated\Endpoint\Payment;
+use bunq\Model\Generated\Endpoint\PaymentBatch;
+use bunq\Model\Generated\Endpoint\RequestInquiry;
+use bunq\Model\Generated\Endpoint\RequestResponse;
+use bunq\Model\Generated\Endpoint\ScheduleInstance;
+use bunq\Model\Generated\Endpoint\SchedulePayment;
+use bunq\Model\Generated\Endpoint\ShareInviteBankInquiry;
+use bunq\Model\Generated\Endpoint\ShareInviteBankResponse;
+use bunq\Model\Generated\Object\NotificationUrl;
+use bunq\test\BunqSdkTestBase;
+use bunq\Util\FileUtil;
+
+/**
+ * Tests:
+ *  NotificationUrl
+ */
+class NotificationUrlTest extends BunqSdkTestBase
+{
+    /**
+     * Getter string constants
+     */
+    const GETTER_PAYMENT = 'getPayment';
+    const GETTER_BUNQ_ME_TAB = 'getBunqMeTab';
+    const GETTER_CHAT_MESSAGE_ANNOUNCEMENT = 'getChatMessageAnnouncement';
+    const GETTER_DRAFT_PAYMENT = 'getDraftPayment';
+    const GETTER_MASTER_CARD_ACTION = 'getMasterCardAction';
+    const GETTER_MONETARY_ACCOUNT_BANK = 'getMonetaryAccountBank';
+    const GETTER_PAYMENT_BATCH = 'getPaymentBatch';
+    const GETTER_REQUEST_INQUIRY = 'getRequestInquiry';
+    const GETTER_REQUEST_RESPONSE = 'getRequestResponse';
+    const GETTER_SCHEDULE_PAYMENT = 'getScheduledPayment';
+    const GETTER_SCHEDULE_INSTANCE = 'getScheduledInstance';
+    const GETTER_SHARE_INVITE_BANK_INQUIRY = 'getShareInviteBankInquiry';
+    const GETTER_SHARE_INVITE_BANK_RESPONSE = 'getShareInviteBankResponse';
+
+    /**
+     * Assertion errors.
+     */
+    const ASSERT_SHOULD_NOT_REACH_THIS_CODE_ERROR = 'Congratulations you\'ve reached unreachable code.';
```

Comment:
Please prefix all the error constants with `ERROR_` (instead of `_ERROR` suffix); Also please put them to the top of the file (since this is how we do it across the SDKs.

Reply:
Also, the text of this assertion doesn't seem to be very descriptive :(